### ISSUE TITLE
User Agent style for href hyperlinks

### ISSUE THREAD

**Issue by fred-wang** [mathml-refresh/mathml#94]
cc @rwlbuis, @bkardell, @emilio

In https://github.com/mathml-refresh/mathml-core/commit/4f34194d9e9be1f0ea679012e4bf2e7b799cde64, I updated a bit the paragraph for href. The suggested style is:

    :any-link {
      color: blue;
      text-decoration: none;
    }

Chromium's one is: 

    :-webkit-any-link {
        color: -webkit-link;
        cursor: pointer;
    }

    :-webkit-any-link:active {
        color: -webkit-activelink;
    }

WebKit's one is 

    :any-link {
      color: -webkit-link;
      cursor: auto;
    }
    :any-link:active {
      color: -webkit-activelink;
    }

Firefox's one is:

    :any-link {
        text-decoration: none !important;
     }

We need consistent style.

**Comment by fred-wang** [comment]
MathML elements with tabIndex are also focusable and it would be good to have style for :focus too.

**Comment by fred-wang** [comment]
I'm adding this one to Gecko (copied from svg):

    *:-moz-focusring {
      /* Don't specify the outline-color, we should always use initial value. */
      outline: 1px dotted;
    }

**Comment by bkardell** [comment]
> We need consistent style.

Do we? Is this true of links in general in the UA sheets?

**Comment by fred-wang** [comment]
> > We need consistent style.
> 
> Do we? Is this true of links in general in the UA sheets?

I don't think this is true in general. But would be nice to make things more consistent if possible :-)

**Comment by NSoiffer** [comment]
If they differ between browsers, should we be consistent with the browser's styling?

**Comment by fred-wang** [comment]
there was no clear consensus, but likely we won't have interoperable for now. would be good to check what html and svg do.

regarding link in math, Murray said office only makes the link blue, without underline

**Comment by NSoiffer** [comment]
At the meeting today, we checked and SVG says very little in their stylesheet about links, definitely not colors. It was also noted that this issue is basically irrelevant now since Level 1 won't likely have links, although focusable elements might be relevant. Brian thought that some tests about that might be relevant.

**Comment by fred-wang** [comment]
@bkardell  So can we go ahead and remove the href attribute and all related link stuff from the current spec?

**Comment by NSoiffer** [comment]
As per the core meeting today, I'm removing the 'need resolution' and other labels and adding 'level 2'. It looks like the spec is up to date.
